operations/firmware/Update_Firmware_with_FAS.md

```diff
@@ -123,6 +124,7 @@ After identifying which hardware is in the system, start with the top item on th
    1. [`iLO 5` (BMC)](FAS_Use_Cases.md#manufacturer-hpe--device-type-nodebmc--target-ilo-5-bmc)
    1. [`iLO 6` (BMC)](FAS_Use_Cases.md#manufacturer-hpe--device-type-nodebmc--target-ilo-6-bmc)
    1. [System ROM (BIOS)](FAS_Use_Cases.md#manufacturer-hpe--device-type-nodebmc--target-system-rom-bios)
+1. [`Foxconn` (Paradise)](FAS_Paradise.md)
```

Comment:
Can remove ` around Foxconn if you add Foxconn to .spelling in the root directory.